src/input.c

```diff
@@ -41,7 +42,12 @@ void input_handle_click(unsigned int button, bool button_down, int mouse_x, int
                         return;
                 }
 
-                if (act == MOUSE_DO_ACTION || act == MOUSE_CLOSE_CURRENT) {
+                if (act == MOUSE_CONTEXT_ALL) {
+                        context_menu();
+                        return;
```

Comment:
I do have a comment on this code though. You should make this a break instead, since that will make sure dunst is waked up after this action (see a few lines below). I'm not sure why it works this way, but it's better to be safe.

Reply:
Fixed in ac7eaf0

Reply:
Great, could you squash that commit onto the commit where you made that code?